main.py

```diff
@@ -1,45 +1,46 @@
 """
-You need to create a github repo (public), plan project structure, use MIT license. 
-Also you need to decide how to run two different conversions with a single script, e.g. how to decide which conversion you have to perform.
+import argparse - For parsing arguments when run script
 
+function def cel_to_fahren(tempreture_in_celsium): - Take temperature in Celsius and convert to Fahrenheit
```

Comment:
@MariaMaximova, ooops, I meant that you need to move this descrtiption not to common three-quoted description but to the function description itself. See an example:
```
def my_function(argument):
    """
    - Take temperature in Celsius and convert to Fahrenheit
    """
    pass
```